### PR TITLE
Add instructions for creating games and test_simple_race

### PR DESCRIPTION
**Pull request by yu239**
More details for creating games. TODO: write C++ test binaries similar to Python test files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,115 @@ To get started, several examples of the simulator Python APIs can be found in
 <xworld_path>/python/examples
 ```
 
+# Instructions for creating games
+Below we explain the flags that the user can set for each game. If you use Python APIs, these flags are passed in as a dictionary when creating the game (see <xworld_path>/python/examples for details). If you use C++ APIs, these flags are either set globally from command line with GFlags, or are passed as arguments of the class constructor (see <xworld_path>/examples for details).
+
+|**ID**|**Python Keyword**|**C++ Class Name**|**Flags**|
+|---|---|-----|-----------|
+|**a**|```simple_game```|```SimpleGame```|```pause_screen```,```array_size```|
+|**b**|```simple_race```|```SimpleRace```|```pause_screen```,```window_width```,```window_height```,```track_type```,```track_width```,```track_length```,```track_radius```,```race_full_manouver```,```random```,```difficulty```|
```
Comment (Haichao-Zhang) [inline README.md]: Is it possible to break the long lines with line wrapping?

File: README.md [hunk 2]
```diff
@@ -81,6 +81,115 @@ To get started, several examples of the simulator Python APIs can be found in
...
+    
+    MALMO mission name, currently only support "demo". (Default: "demo")
+
+* ```minecraft_client_ip``` (**e**)
+    
+    IP address of Minecraft mod. (Default: "127.0.0.1")
+
+* ```minecraft_client_port``` (**e**)
+    
+    pport number of Minecraft mod. (Default: 10000)
```
Comment (Haichao-Zhang) [inline README.md]: pport -> port

File: README.md (hunk 1 above)
Comment (yu239) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (yu239) [reply README.md]: Done

**Pull request merged**